controllers/front/ProductController.php

```diff
@@ -710,7 +711,7 @@ protected function formTargetFormat()
         $this->context->smarty->assign('customizationFormTarget', $customization_form_target);
     }
 
-    protected function formatQuantityDiscounts($specific_prices, $price, $tax_rate, $ecotax_amount)
```

Comment:
You're right, there is some compatibility break. My rationale was: it is a protected function, then its signature can be changed. I just forgot inheritance, and especially overrides.
So, $price parameter could be kept as unused.
But anyway, first parameter $specific_prices should include additional base_price field in order the new formatting succeeds